models/review.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose')
+
+const { ObjectId } = mongoose.Schema
+
+const reviewSchema = new mongoose.Schema({
+  userId: { type: ObjectId },
+  businessId: { type: ObjectId },
+  comment: { type: String },
+  rating: { type: Number, required: true },
+  // createDate: { type: Date, default: Date.now },
```

Comment:
Hi @cynthiaeddy! There may be value in tracking the date a review is added to the system, especially if it is a bad review that negatively impacts a business. Did you plan on bringing this back in later once you finalized the feature?

Reply:
yup, planning on bringing it back